src/database/migrations/20210708145433-change-student-field.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = {
+  up: async (queryInterface, Sequelize) => {
+    return queryInterface.sequelize.transaction((t) => {
+      return Promise.all([
+        queryInterface.removeColumn('Students', 'stream'),
+        queryInterface.addColumn('Students', 'streamId', {
+          allowNull: false,
+          type: Sequelize.INTEGER,
+        }),
```

Comment:
В запросы нужно передать параметром транзакцию 
https://sequelize.org/master/class/lib/transaction.js~Transaction.html
```suggestion
    return queryInterface.sequelize.transaction((t) => {
      return Promise.all([
        queryInterface.removeColumn('Students', 'stream' ,{ transaction: t }),
        queryInterface.addColumn('Students', 'streamId', {
          allowNull: false,
          type: Sequelize.INTEGER,
          },
         { transaction: t }),
```